scipy-1.0/paper.tex

```diff
@@ -263,6 +263,26 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+The \texttt{scipy.stats} module provides a framework to implement probability
+distributions.  Given an expression for a probability distribution function (pdf),
```

Comment:
@mdhaber both. Users can implement their own, but it's not that common because so many distributions are already built-in.

@ev-br thanks, I'll take that over.